src/components/button.jsx

```diff
@@ -0,0 +1,17 @@
+function ButtonComponent(props) {
```

Comment:
It would be better if you specified a clear description of the props you are using. Ex: buttonProps.
Or even use destructuring